libs/design/src/molecules/modal/modal.ts

```diff
@@ -2,7 +2,12 @@ import { ComponentRef } from '@angular/core';
 import { DaffModalComponent } from './modal/modal.component';
 import { OverlayRef } from '@angular/cdk/overlay';
 
-export interface DaffModal {
+export class DaffModal<T> {
+
```

Comment:
This function exists on the modal service. This is really more of a helper function. I don't think we should be providing two ways of doing the same thing.

Reply:
changed in 93fdd37ad40ae5d86f9f21688d883ca8854b93aa. We could potentially add a directive to go on a button, similar to material, that would close the modal.